src/hyperloglog.c

```diff
@@ -1041,6 +1045,121 @@ int hllAdd(robj *o, unsigned char *ele, size_t elesize) {
     }
 }
 
+#if defined(HAVE_AVX2)
+/* Optimized for the default configuration: 16384 registers of 6 bits each. */
```

Comment:
maybe we should add more detail comments for these functions, including dense representation, AVX2 keywords

Reply:
changed in 0bae63f62516ede32b8d3c023eadd61152e12117